### PR TITLE
Supported small size (M,N,K<8)  and added to gtest for F8/BF8 in/out combinations

### PR DESCRIPTION
**Pull request by solaslin**
- [x] Updated logic yamls for eight combinations A/B/D = [F8/BF8]
- [x] Added and passed the small size gtests for all combinations
- [x] Each logic yaml contains new solutions for problems: [3,3,1,3] and [1,1,1,128] both

### CODE REVIEW

File: clients/gtest/matmul_gtest.yaml [hunk 1]
```diff
@@ -736,7 +734,7 @@ Tests:
   bias_vector: [0, 1]
   bias_type: f32_r
   unit_check: 1
-  gpu_arch: '942'
+  gpu_arch: '94[0-2]'
```
Comment (jichangjichang) [inline clients/gtest/matmul_gtest.yaml]: Don't need to update 940 nd 941 logic yaml and test.
942 is enough.

**Review by KKyang — DISMISSED** [review]

File: clients/gtest/matmul_gtest.yaml (hunk 1 above)
Comment (solaslin) [reply clients/gtest/matmul_gtest.yaml]: OK, done!!

**Review by jichangjichang — APPROVED** [review]
please update local full hipblaslt-test results

**Comment by solaslin** [comment]
> please update local full hipblaslt-test results

Sure, i'm running it now

**Comment by solaslin** [comment]
local full gtest:

[----------] Global test environment tear-down
[==========] 48206 tests from 13 test suites ran. (2037932 ms total)
[  PASSED  ] 48206 tests.
hipBLASLt version: 1000

command line: ./hipblaslt-test

**Review by hcman2 — APPROVED** [review]

**Pull request merged**